### PR TITLE
Add pykale description

### PR DESCRIPTION
**Pull request by bobturneruk**

### CODE REVIEW

File: _data/projects.csv [hunk 1]
```diff
@@ -14,7 +14,7 @@ id,key,title,long_title,tech_methods,rses,start,end,department,level,show
 17,,ACCE DTP,ACCE DTP,,Anna Krystalli,01/01/2020,30/04/2020,University of York,2,1
 19,,Reprohacks,Reprohacks,,Anna Krystalli,01/01/2020,31/03/2020,N8 ,2,1
 21,,RateSetter+,RateSetter+,,John Charlton,18/11/2019,17/03/2021,Mechanical Engineering,2,1
-22,,Prognostic: Cardiac MRI with ML,Prognostic: Cardiac MRI with ML,"python, version control, static analysis, GitHub","David Jones, Robert Turner, Will Furnass",01/10/2020,30/09/2021,Computer Science,2,1
+22,pykale,Prognostic: Cardiac MRI with ML,pykale (Knowledge Aware Machine Learning),"python, version control, static analysis, GitHub, testing, machine learning","David Jones, Robert Turner, Will Furnass",01/10/2020,30/09/2021,Computer Science,1,1
```
Comment (haipinglu) [inline _data/projects.csv]: Suggestion:
- title (2 alternatives)
  - Knowledge-aware machine learning
  - Knowledge-aware learning
- long_title (3 alternatives)
  - PyTorch library for multimodal/transfer learning on graphs/images/videos 
  - Multimodal/transfer learning on graphs/images/videos 
  - PyTorch library for multimodal/transfer learning

Comment (bobturneruk) [reply _data/projects.csv]: Thanks @haipinglu - the title is a link back to another database so won't be displayed (and I can't change).

File: _project_descriptions/pykale.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+key: pykale
+---
+
+[pykale](https://github.com/pykale/pykale) is a python package aimed at making it easier for people to access machine learning. An important target audience are clinicians and clinical researchers who have amenable data to analyse, but lack the time and expertise to apply machine learning approaches. The work is supported by the [Wellcome Trust](https://wellcome.org/grant-funding/schemes/innovator-awards-digital-technologies).
```
Comment (willfurnass) [inline _project_descriptions/pykale.md]:
```suggestion
[pykale](https://github.com/pykale/pykale) is a Python package aimed at making it easier for people to access machine learning. An important target audience are clinicians and clinical researchers who have amenable data to analyse, but lack the time and expertise to apply machine learning approaches. The work is supported by the [Wellcome Trust](https://wellcome.org/grant-funding/schemes/innovator-awards-digital-technologies).
```

File: _project_descriptions/pykale.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+RSE involvement in the project supports use of version control and collaboration via GitHub, as well as static analysis, release mechanism, packaging and software testing.
```
Comment (willfurnass) [inline _project_descriptions/pykale.md]: supports _the_ use of

Comment (willfurnass) [inline _project_descriptions/pykale.md]: Suggest rewording to say developed/helped develop strategies for x, y, z (or could say advised on those things).  Project description could be used to illustrate what a 10% advisory/consultant-like involvement in a project looks like.

**Review by willfurnass — APPROVED** [review]
Think nature of contributions could be rephased but otherwise LGTM.

File: _project_descriptions/pykale.md (hunk 2 above)
Comment (bobturneruk) [reply _project_descriptions/pykale.md]: Thanks!

File: _project_descriptions/pykale.md (hunk 3 above)
Comment (bobturneruk) [reply _project_descriptions/pykale.md]: Thanks!

Comment (bobturneruk) [reply _project_descriptions/pykale.md]: Good point! I hope I've caught this in the latest version.

**Comment by bobturneruk** [comment]
@haipinglu has further comment when he gets time, so please don't merge.

File: _project_descriptions/pykale.md (hunk 3 above)
Comment (willfurnass) [reply _project_descriptions/pykale.md]: Def better!

File: _project_descriptions/pykale.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+key: pykale
+---
+
+[pykale](https://github.com/pykale/pykale) is a ([pytorch](https://pytorch.org/)-based) Python package aimed at making it easier for people to access machine learning. An important target audience are clinicians and clinical researchers who have amenable data to analyse, but lack the time and expertise to apply machine learning approaches. The work is supported by the [Wellcome Trust](https://wellcome.org/grant-funding/schemes/innovator-awards-digital-technologies).
```
Comment (haipinglu) [inline _project_descriptions/pykale.md]: The primary motivation is still for research so I would like to revise this to something like below (feel free to revise further):

[PyKale](https://github.com/pykale/pykale) is a ([PyTorch](https://pytorch.org/)-based) Python package aimed at standardising machine learning workflows for graphs, images, and videos with a unified pipeline-based API to accelerate cross-disciplinary research. This package also makes it easier for end-users to access machine learning. An important target audience are clinicians and clinical researchers who have abundant data to analyse, but lack the time and expertise to apply machine learning approaches. The work is supported by the [Wellcome Trust](https://wellcome.org/grant-funding/schemes/innovator-awards-digital-technologies).

File: _project_descriptions/pykale.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+
+A key RSE involvement in the project to date was the development of a documented strategy to add automated testing to the package. We also advise on the use of version control and collaboration via GitHub, as well as static analysis, release mechanisms and packaging.
+
+Lack of expertise and governance barriers to transferring data from clinical organisations can mean that adoption of machine learning is reduced. Machine learning has the potential to improve diagnosis, prognosis and selection of treatment, improving health and quality of life for patients. A key example is analysis of medical imaging data to identify unusual features that may be of clinical interest.
```
Comment (haipinglu) [inline _project_descriptions/pykale.md]: I suggest the following rearrangement and addition to close this description:

Machine learning has the potential to improve diagnosis, prognosis and selection of treatment, improving health and quality of life for patients. A key example is analysis of medical imaging data to identify unusual features that may be of clinical interest. Lack of expertise and governance barriers to transferring data from clinical organisations can mean that adoption of machine learning is reduced. This easily-accessible machine learning package will bring machine learning software closer to clinicians for improved clinical outcomes powered by AI.

**Review by haipinglu — APPROVED** [review]

File: _project_descriptions/pykale.md (hunk 4 above)
Comment (bobturneruk) [reply _project_descriptions/pykale.md]: Thank you!

File: _project_descriptions/pykale.md (hunk 5 above)
Comment (bobturneruk) [reply _project_descriptions/pykale.md]: Thanks!
